conf/messages.en

```diff
@@ -598,6 +598,36 @@ constructionalEngineeringList3.occupation2 = labourers
 constructionalEngineeringList3.occupation3 = shop-helpers
 constructionalEngineeringList3.occupation4 = slewers and straighteners
 
+shipyardOccupationList1.title = Shipyard occupation List 1 of 2
+shipyardOccupationList1.heading = Shipyard occupation List 1 of 2
+shipyardOccupationList1.checkYourAnswersLabel = Shipyard occupation List 1 of 2
+shipyardOccupationList1.error.required = Select yes if your occupation is listed
+shipyardOccupationList1.radioLabel = Is your occupation listed?
+shipyardOccupationList1.listText = The occupations are:
+shipyardOccupationList1.occupation1 = blacksmiths and their strikers
+shipyardOccupationList1.occupation2 = boilermakers
+shipyardOccupationList1.occupation3 = burners
+shipyardOccupationList1.occupation4 = carpenters
+shipyardOccupationList1.occupation5 = caulkers
+shipyardOccupationList1.occupation6 = drillers
+shipyardOccupationList1.occupation7 = furnacemen (platers) holders up
+
+shipyardOccupationList2.heading = Shipyard Occupation List 2 of 2
+shipyardOccupationList2.checkYourAnswersLabel = Shipyard Occupation List 2 of 2
+shipyardOccupationList2.error.required = Select yes if shipyardOccupationList2 of 2
+shipyardOccupationList2.error.required = Select yes if your occupation is listed
```

Comment:
Two error.required? Think this should just be the "shipyardOccupationList2.error.required = Select yes if your occupation is listed" option